config/locales/de.yml

```diff
@@ -2315,6 +2315,21 @@ de:
     no_old_ones: Es wurde noch nichts abgegeben.
     no_future_ones: >
       Es sind keine Hausauafgaben jenseits der aktuellen vorhanden.
+    no_manuscript_yet: Es wurde noch keine Datei hochgeladen.
+    correct_file_size: Die Datei hat die richtige Größe.
+    file_size_too_big: Die Datei ist eigentlich zu groß, kann aber hochgeladen werden.
+    file_size_way_too_big:  Die Dateigröße überschreitet das Limit. Bitte optimiere sie.
+    optimization_help: >
+      <b>Optimieren:</b>Verwende diesen Knopf, um automatisch 
+      im Browser eine Optimierung durchzuführen. Dies dauert 1-2 Minuten.
+    optimization_info: >
+      Dazu wird ein kleines Zusatzprogramm heruntergeladen und <i>im Browser</i> ausgeführt.
+       So bleiben die Daten auf deinem Gerät, bis Du sie hochgeladen hast.
+    assure_third_party: >
+      Ich versichere, dass durch die Datei keine Rechte Dritter verletzt werden.
+    removal_notice: >
+      <b>Hinweis:</b> Dateien werden nach Ablauf des Semesters gelöscht.
+      Wir übernehmen keine Haftung für mögliche Datenverluste.
```

Comment:
See the remarks about missing translations in the review for the submissions form partial.